site/dat/docs/Apiritif.md

```diff
@@ -824,6 +824,10 @@ These actions require a value parameter, the possible values are:
   - `win\_ser\_local`: Go to the initial window.
   - `no value`: When no value is assigned, it means that the selection action is assigned over the last created window, and if the close action is used, it will also be over the last one created.
 
+Note: When any action command opens a new window (like click on a link with target window assigned), 
+the action of selecting the window must always be declared, otherwise the actions executed by the execution 
```

Comment:
i think this line is no longer correct, if we auto-switch after opening window

Reply:
Actually we just auto switch if there is an explicit action 'openWindow' but in case a link opens a new window we are no longer detecting it and the switchWindows action needs to be declared...